src/libs/NetworkConnection.ts

```diff
@@ -68,46 +69,58 @@ Onyx.connect({
 });
 
 /**
- * Set up the event listener for NetInfo to tell whether the user has
- * internet connectivity or not. This is more reliable than the Pusher
- * `disconnected` event which takes about 10-15 seconds to emit.
+ * Set interval to periodically (re)check backend status
+ * @returns for use with clearInterval
  */
-function subscribeToNetInfo(): void {
-    // Note: We are disabling the configuration for NetInfo when using the local web API since requests can get stuck in a 'Pending' state and are not reliable indicators for "offline".
-    // If you need to test the "recheck" feature then switch to the production API proxy server.
-    if (!CONFIG.IS_USING_LOCAL_WEB) {
-        // Calling NetInfo.configure (re)checks current state. We use it to force a recheck whenever we (re)subscribe
-        NetInfo.configure({
-            // By default, NetInfo uses `/` for `reachabilityUrl`
-            // When App is served locally (or from Electron) this address is always reachable - even offline
-            // Using the API url ensures reachability is tested over internet
-            reachabilityUrl: `${CONFIG.EXPENSIFY.DEFAULT_API_ROOT}api/Ping`,
-            reachabilityMethod: 'GET',
-            reachabilityTest: (response) => {
+function subscribeToBackendReachability() {
+    return setInterval(() => {
```

Comment:
Maybe we could return a function like...

```ts
const intervald = setInterval(...);

return () => {
  clearInterval(intervald);
};
```

We'd call our util like this...

```ts
const unsubscribeFromBackendReachability = !CONFIG.IS_USING_LOCAL_WEB ? subscribeToBackendReachability() : undefined;
```

This would be symmetric with how we handle NetInfo, and also cleanly abstract the specific API we use in the implementation.

Reply:
`unsubscribeFromBackendReachability` could be called like this:

```ts
unsubscribeFromBackendReachability?.();
```